Source/Bugsnag.m

```diff
@@ -54,6 +57,7 @@ + (void)startBugsnagWithConfiguration:(BugsnagConfiguration *)configuration {
         if ([configuration hasValidApiKey]) {
             bsg_g_bugsnag_notifier =
                     [[BugsnagNotifier alloc] initWithConfiguration:configuration];
+            [self startPlugins];
```

Comment:
Does it make sense to initialise the plugins before the notifier has been initialised? My first thought is that it would make more sense to start the plugins after `[bsg_g_bugsnag_notifier start]` has been invoked, as this could lead to unintended behaviour if the plugin believes the notifier is already initialised. I'm unsure _why_ we need a plugin interface so it'd be good to understand more about that too.

Reply:
If its initialized _after_ start, then reports could be sent or breadcrumbs saved, etc, before the plugin has a chance to modify the configuration or client instance.